src/psyclone/psyir/symbols/symbol_table.py

```diff
@@ -933,32 +939,37 @@ def specify_argument_list(self, argument_symbols):
         self._validate_arg_list(argument_symbols)
         self._argument_list = argument_symbols[:]
 
-    def lookup(self, name, visibility=None, scope_limit=None):
+    def lookup(self, name, visibility=None, scope_limit=None,
+               default=DEFAULT_SENTINEL):
```

Comment:
Ah, I was under the mistaken impression that `default` was what it was called in `dict.get()`. I agree that `value` is a rubbish name. I've gone for `otherwise`.